index.html

```diff
@@ -0,0 +1,192 @@
+				<div class="header__nav--logo">
+					<h1>nova</h1>
+				</div>
+				<div class="header__nav--menu" id="#" <a href="#"></a>
+					<a href="#team">The Team</a>
+					<a href="#pricing">Pricing</a>
+					<a href="#features">Features</a>
+				</div>
+			</nav>
+			<div class="header__front wrapper">
+				<div class="header__front--text">
+					<h2>Beautiful Free Nova template</h2>
+					<p>
+						Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsam
+						exercitationem repellendus beatae!
+					</p>
+					<button class="header__front--btn">Download FREE</button>
+				</div>
+			</div>
+			<div class="header__front wrapper">
+				<div class="header__front--img"></div>
+			</div>
+			<div class="header__front--arrow-down">
+				<a href="#features" id="#features"<i class="fa-solid fa-arrow-down"></i></a>
+			</div>
+		</header>
+
+		<main id="features" class="main">
+			<div class="main__features wrapper">
+				<h2>features</h2>
+			</div>
+			<div class="main__features--list wrapper">
+				<div class="main__features--list-one display-flex">
+					<i class="fa-solid fa-phone icon-list"></i>
+					<h3>fully responsive</h3>
+					<p>
+						Lorem ipsum dolor sit amet consectetur adipisicing elit. Vero
+						sapiente, in rerum cumque voluptatum voluptatibus.
+					</p>
+				</div>
+				<div class="main__features--list-two display-flex">
+					<i class="fa-solid fa-screwdriver-wrench icon-list"></i>
+					<h3>castomizable</h3>
+					<p>
+						Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel optio
+						deleniti porro eligendi, impedit quasi! Lorem ipsum dolor sit amet.
+					</p>
+				</div>
+				<div class="main__features--list-three display-flex">
+					<i class="fa-solid fa-medal icon-list"></i>
+					<h3>slick and beautiful disign</h3>
+					<p>
+						Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore vero
+						voluptas debitis ducimus minima fugit.
+					</p>
+				</div>
+			</div>
+
+			<div class="logos">
+				<div class="logos__img wrapper"></div>
+			</div>
+
+			<div class="apple">
+				<div class="apple__img"></div>
+				<div class="apple__info">
+					<h3>Simple and beautiful</h3>
+					<p>
+						Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+						Architecto, alias. Necessitatibus, voluptates iure. Rem, dolore.
+						Corporis, doloribus beatae perferendis voluptate facilis ut ipsum
+						dolore vel reprehenderit cupiditate vitae reiciendis atque enim
+						aperiam sunt ex. Numquam voluptas adipisci maxime modi ratione?
+						Aliquid eum consequatur mollitia ipsam laudantium facilis minima
+						natus magni.
+					</p>
+				</div>
+			</div>
+
+			<section id="pricing" class="pricing">
+				<div class="pricing__info wrapper">
+					<h3>choose your pricing plan</h3>
+					<p>
+						Lorem ipsum, dolor sit amet consectetur adipisicing elit. Itaque,
+						facilis.
+					</p>
+				</div>
+
+				<div class="pricing__offers wrapper">
+					<div class="pricing__offer">
```

Comment:
Sporo jest div-ów więc zrobiłbym `article` dla każdego pakietu cenowego, bo to oddzielna, jasna spójna całość.